src/errors/runtime/index.js

```diff
@@ -28,11 +28,15 @@ export class GeneralError extends Error {
 
 export class TestCompilationError extends Error {
     constructor (originalError) {
-        const template = TEMPLATES[RUNTIME_ERRORS.cannotPrepareTestsDueToError];
+        const template     = TEMPLATES[RUNTIME_ERRORS.cannotPrepareTestsDueToError];
+        const errorMessage = originalError.toString();
 
-        super(renderTemplate(template, originalError.toString()));
+        super(renderTemplate(template, errorMessage));
 
-        Object.assign(this, { code: RUNTIME_ERRORS.cannotPrepareTestsDueToError });
+        Object.assign(this, {
+            code: RUNTIME_ERRORS.cannotPrepareTestsDueToError,
+            data: [ { errorMessage } ]
```

Comment:
I think we can use `errorMessage` in data as string instead an object

Reply:
Ok